website/docs/r/eventgrid_event_subscription.html.markdown

```diff
@@ -127,6 +127,14 @@ A `webhook_endpoint` supports the following:
 
 * `url` - (Required) Specifies the url of the webhook where the Event Subscription will receive events.
 
+* `max_events_per_batch` - (Optional) Maximum number of events per batch.
```

Comment:
can we document the `base_url` field here too?

Reply:
done